pkgs/development/ruby-modules/gem-config/default.nix

```diff
@@ -69,7 +69,8 @@ in
   };
 
   capybara-webkit = attrs: {
-    buildInputs = [ qt48 ];
+    buildInputs = [ qt48 ] ++ stdenv.lib.optionals stdenv.isDarwin [ darwin.apple_sdk.frameworks.Cocoa ];
+    NIX_CFLAGS_COMPILE = stdenv.lib.optionalString stdenv.isDarwin "-I${libcxx}/include/c++/v1";
```

Comment:
libcxx is part of the darwin stdenv, I would expect it to work without this.

Reply:
Seems to be required in more places than just here: 
https://github.com/NixOS/nixpkgs
https://github.com/NixOS/nixpkgs/search?utf8=%E2%9C%93&q=nix_cflags_compile+isdarwin+libcxx&type=

Reply:
For llvm it's expected since we want the version that matches the llvm version we're building, the others I don't understand.

Reply:
In Python it might be a similar problem.

Reply:
Oh, that could explain it. What do I build to test this?

Reply:
```
# Gemfile
source "https://rubygems.org"
gem "capybara-webkit"
```

Then run `bundle` from the `bundler` package followed by `bundix` (also in nixpkgs).

Reply:
Did you manage to test this @LnL7 ?

Reply:
@alyssais I suppose you have tested this? Then I could merge it as well.

Reply:
Yep! Been using it successfully for weeks. Without this change it didn't work at all.

Reply:
Yeah sorry about the delay, I forgot about it because of a long qt build.

@Mic92 Good point about `CC`/`CXX`, it's probably a similar problem as we've seen with python.